docs/about.md

```diff
@@ -11,10 +11,18 @@ Unlike other ingress controllers, Envoy supports continual reconfiguration witho
 
 ## How Contour works
 
+Contour supports both Ingress objects and IngressRoute objects which are ways to define traffic routes into a cluster. 
+They are different in how the objects are structured as well as how they are implemented, however, are identical at their core intent, to simply route traffic. 
```

Comment:
nit: the intent of the Ingress and IngressRoute objects is to configure the routing of ingress traffic, not to actually route traffic.

Reply:
:+1: